### ISSUE TITLE
Single-line splitting mode is joining sentence across lines

### ISSUE THREAD

**Issue by yucongo** [fnl/segtok#11]
In [37]: from segtok.segmenter import split_single

    In [38]: list(split_single('Folding Beijing\nby Hao Jingfang'))
    Out[38]: ['Folding Beijing\nby Hao Jingfang']

I  can't figure out why this one does not split into two (I would expect ['Folding Beijing', 'by Hao Jingfang']). list(split_single('a\nb\nc')) seems to split OK with ['a', 'b', 'c'] as result.

Great work! Thanks a lot. nltk and textblob dont play well with cxfreeze, giving me a headache...

**Comment by fnl** [comment]
Sounds very much like a bug. Will take a look and fix asap. Thanks for the nice words, glad this little tool is useful.

**Comment by fnl** [comment]
Turns out, single-line splitting mode was indeed joining lines when the next line started with continuation words that hardly ever, if at all, indicate line breaks (like, here, "by ...").

I've fixed segtok to be more rigorous about this and not leave it any room for *not* splitting on every line in single line splitting mode or when using the split_single API function (see c572742), so that should fix this particular case as well as any possibly related ones. I've bumped the version and [released](https://pypi.python.org/pypi/segtok/1.5.6) an update to PyPI (see dc1f8e1 - new version: 1.5.6).
